datafusion/sqllogictest/test_files/string/dictionary_utf8.slt

```diff
@@ -37,6 +37,52 @@ select arrow_cast(col1, 'Dictionary(Int32, Utf8)') as c1 from test_substr_base;
 statement ok
 drop table test_source
 
+# TODO: move it back to `string_query.slt.part` after fixing the issue
```

Comment:
This experience moving tests has found many great bugs. Thank you @goldmedal for finding and filing them